go.mod

```diff
@@ -43,16 +43,16 @@ replace (
 	github.com/evanphx/json-patch => github.com/evanphx/json-patch v0.0.0-20191220184958-bf22ed931162
 	github.com/fsnotify/fsnotify => github.com/fsnotify/fsnotify v1.4.7
 	github.com/gin-contrib/sse => github.com/gin-contrib/sse v0.0.0-20170109093832-22d885f9ecc7
-	github.com/gin-gonic/gin => github.com/gin-gonic/gin v0.0.0-20180501062418-bd4f73af679e
+	github.com/gin-gonic/gin => github.com/gin-gonic/gin v1.8.2
```

Comment:
I believe this is a different commit from previous (v1.8.2 is `6a2a260`), which should require updating DEPENDENCIES.md